### PR TITLE
Add exit checkin to ops

### PR DESCRIPTION
**Pull request by ptulugu**

### CODE REVIEW

**Review by jkilpatr — APPROVED** [review]
I've left a few comments but this is overall a great design and properly implemented.

File: althea_types/src/interop.rs [hunk 1]
```diff
@@ -701,6 +701,29 @@ pub struct OperatorCheckinMessage {
     pub rita_uptime: Duration,
 }
 
+/// The message and exit sends to the operator server to checkin, this allows us to customize
+/// the operator checkin response to the device
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct OperatorExitCheckinMessage {
+    pub id: Identity,
```
[the remark this answers is not in the document]
Comment (ptulugu) [reply althea_types/src/interop.rs:698]: fixed

File: scripts/build_exit.sh [hunk 2]
```diff
@@ -4,5 +4,5 @@ cd cross-builders/exit
 docker build -t cross-with-clang-ssl .
 cp Cross.toml ../..
 cd ../..
-cross build --release --target x86_64-unknown-linux-gnu -p rita_bin --bin rita_exit
+cross build --release --target x86_64-unknown-linux-gnu --features rita_bin/exit_development -p rita_bin --bin rita_exit
```
[the remark this answers is not in the document]
Comment (ptulugu) [reply scripts/build_exit.sh]: fixed

File: rita_exit/src/operator_update/update_loop.rs [hunk 3]
```diff
@@ -0,0 +1,51 @@
+//! Independent loop for operator updates, this prevents errors in the rita fast loop from causing the
+//! router to become unresponsive to updates or reboot instructions
```
[the remark this answers is not in the document]
Comment (ptulugu) [reply rita_exit/src/operator_update/update_loop.rs]: fixed

**Pull request merged**